src/cryptography/hazmat/bindings/openssl/ocsp.py

```diff
@@ -0,0 +1,31 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+INCLUDES = """
+/* Windows be crazy */
```

Comment:
This comment does not tell me nothing new.
Maybe it should be left out or updated to describe the original error which required this change.
Thanks!

But... do we need binding for OCSP ?

OSCP is just the C representation of DER encoded OSCP objects.

I think that we can use asn1  to decode/encode this in python.

What do you think?